terraform/gcloud/environments/compute/alternate_volumes.sh

```diff
@@ -6,7 +6,7 @@ MNT_DIR=/pgdata
 
 if [[ -d "$MNT_DIR" ]]; then
    exit
-else 
+else
```

Comment:
Do you think we should change ext4 to XFS?

Reply:
Yes, I do, but not through this PR. Will do it later.

Reply:
Thank you!